tox.ini

```
@@ -0,0 +1,127 @@
+    lint: pre-commit
+    compile: sphinx-theme-builder[cli]
+    profile-docs: py-spy
+extras = 
+    {docs-no-checks, docs-linkcheck, profile-docs}: doc
+skip_install = 
+    lint: true # do not need to install to lint
+    profile-docs: true # avoids issues with py-spy and setting the git repo
+    compile: false
+package = editable
+commands = 
+    lint: pre-commit run -a
+    compile: stb compile
+    # can substitute the target directory
+    docs-no-checks: sphinx-build {posargs:audit}/site {posargs:audit}/_build
+    docs-linkcheck: sphinx-build -W -b linkcheck docs/ docs/_build/html --keep-going
+    # example tox run -e py39-profile-docs -- -o profile.svg -n 100
+    profile-docs: python ./tools/profile.py {posargs}
+
+# tests can be ran with or without coverage (see examples below), it is recommended to run compile before running tests (see examples below), 
+# tox run -e compile,py39-tests 
+# if you want to skip the assets compilation step you can run the tests without `compile`
+# tox run -e py39-tests 
+# run tests with a specific Sphinx version
+# tox run -e compile,py39-sphinx61-tests
+# run tests without coverage
+# tox run -e compile,py39-tests-no-cov
+[testenv:py3{9,10,11,12}{,-sphinx61,-sphinxdev,}-tests{,-no-cov}]
+description = "Run tests Python and Sphinx versions. If a Sphinx version is specified, it will use that version vs the default in pyproject.toml"
+# need to ensure the package is installed in editable mode
+package = editable
+extras =
+    test # install dependencies - defined in pyproject.toml
+deps = 
+    coverage[toml]
+    py39-sphinx61-tests: sphinx~=6.1.0
+    py312-sphinxdev: sphinx[test] @ git+https://github.com/sphinx-doc/sphinx.git@master
+depends = compile
+commands = 
+    py3{9,10,11,12}{,-sphinx61,-sphinxdev,}-tests: coverage run -m pytest -m "not a11y" {posargs}
+    py3{9,10,11,12}{,-sphinx61,-sphinxdev,}-tests-no-cov: pytest -m "not a11y" {posargs}
+
+# run accessibility tests with Playwright and axe-core
+# compiling the assets is needed before running the tests
+# tox run -e compile,py312-docs,a11y-tests
+[testenv:a11y-tests]
+description = run accessibility tests with Playwright and axe-core
+base_python = py312 # keep in sync with tests.yml
+extras = 
+    test
+    a11y
+depends = 
+    compile, 
+    py312-docs
+allowlist_externals=playwright
+commands = 
+    playwright install 
+    pytest -m "a11y" {posargs}
+
+# build PST documentation with the default or a specific Sphinx version
+# since we are building the documentation we install the packaged dev version of PST
+# tox run -e py39-docs
+# tox run -e py39-sphinx61-docs
+[testenv:py3{9,12}{,-sphinx61}-docs]
+description = build the documentation and place in docs/_build/html
+# suppress Py3.11's new "can't debug frozen modules" warning
+set_env = PYDEVD_DISABLE_FILE_VALIDATION=1
+# keep this in sync across all docs environments
+extras = {[testenv:docs-no-checks]extras}
+deps = 
+    py39-sphinx61-docs: {py39-sphinx61-tests[deps]}
```

```suggestion
    py39-sphinx61-docs: py39-sphinx61-tests[deps]
```

I think this is the issue as the error message is

/opt/hostedtoolcache/Python/3.9.19/x64/bin/uv pip install './{py39-sphinx61-tests[deps]}'
error: Failed to read from the distribution cache
  Caused by: failed to query metadata of file `/home/runner/work/pydata-sphinx-theme/pydata-sphinx-theme/{py39-sphinx61-tests[deps]}`